src/main/java/uk/gov/hmcts/reform/ccd/client/model/AboutToStartOrSubmitCallbackResponse.java

```diff
@@ -10,6 +11,7 @@
 
 @Data
 @Builder
+@AllArgsConstructor
```

Comment:
Is this needed? I think https://projectlombok.org/features/experimental/Jacksonized should be added